docs/opensearch-sink-connector-config-options.rst

```diff
@@ -167,6 +158,25 @@ Data Conversion
   * Valid Values: [ignore, warn, fail]
   * Importance: low
 
+Data Stream
+^^^^^^^^^^^
+
+``data.stream.name``
```

Comment:
I think we may need to split this property in 2:

```
data.stream.enabled = {true | false}
data.stream.prefix = (optional) ""
```

Why: it may not be obvious that just by adding `data.stream.name` all indices suddenly become data streams. The `data.stream.enabled` would make it explicit, plus the `data.stream.prefix` (old `data.stream.name`) becomes optional - the data stream name could be the same as index name. 

Thoughts?

Reply:
Well looks a bit better.